### ISSUE TITLE
Question about using other datasets

### ISSUE THREAD

**Issue by lux0818** [librahu/HERec#13]
Hello professor,thanks for your code.
I have a question is that ,when I use DB movie datasets, I change the metapaths ,but when I run HERec ,it has keyError, so, how to config the 'ratedim','itemdim' and 'userdim'? should I change it when I use a bigger dataset?

**Comment by librahu** [comment]
Of course. 'ratedim','itemdim' and 'userdim' are the hyper-parameters.

**Comment by lux0818** [comment]
Thanks for your reply!
can you tell me how to change it when I use Douban movies ande Douban music datasets?

**Comment by librahu** [comment]
They are tuned via the performance.
